### PR TITLE
Fix TransportShardUpsertActionTest

### PR DESCRIPTION
**Pull request by matriv**

### CODE REVIEW

File: es/es-repository-hdfs/build.gradle [hunk 1]
```diff
@@ -0,0 +1,26 @@
+dependencies {
```
Comment (seut) [inline es/es-repository-hdfs/build.gradle]: how is this related to the commit msg?

Comment (matriv) [inline es/es-repository-hdfs/build.gradle]: omg ... I accidentally pushed the other files
